subsys/zigbee/lib/zigbee_app_utils/Kconfig

```diff
@@ -12,6 +12,8 @@ menuconfig ZIGBEE_APP_UTILS
 	  - for parsing and converting Zigbee data types
 	  - for handling common ZBOSS signals, with behavior depending on the device role
 	  - for indicating the device status at the network using onboard LEDs
+	# Factory Reset uses DK buttons
+	imply DK_LIBRARY
```

Comment:
I'm not sure if that's a good idea. Can we add an additional Kconfig like `ENABLE_FACTORY_REASET`, set it's default value as `y` and imply it from there?

Consider the case, when someone would like to use the default signal handler, but his custom board does not have buttons, nor implements the `DK_LIBRARY` handlers.

Reply:
@tomchy Sounds reasonable